src/Components/ViewColleaguePage/index.jsx

```diff
@@ -1,12 +1,12 @@
 import { memo } from 'react';
 import { Header } from 'semantic-ui-react';
 
-const ViewColleague = memo(() => (
+const ViewColleaguePage = memo(() => (
```

Comment:
It is already known that it is a page. 
```suggestion
const ViewColleague = memo(() => (
```